default.yml

```diff
@@ -66,6 +50,10 @@ Metrics/MethodLength:
 Metrics/ClassLength:
   Max: 1500
 
+# Exclude RSpec
+Metrics/BlockLength:
+  ExcludedMethods: ['describe', 'context']
```

Comment:
This allows large blocks in specs, so we can stop disabling this rule on every spec.

Reply:
+1

Reply:
+:100:

Reply:
👍

Reply:
OMG yes.